man/rudder.8

```diff
@@ -0,0 +1,286 @@
+\fBrudder agent update\fR
+before to update the promises\&.
+.sp
+\fBOptions\fR:
+.sp
+\fB\-i\fR: run the agent in information mode, prints basic information
+.sp
+\fB\-v\fR: run the agent in verbose mode, prints detailed information
+.sp
+\fB\-d\fR: run the agent in debug mode, prints low\-level information
+.sp
+\fB\-q\fR: run the agent in quiet mode (display only error messages)
+\fB\-w\fR: show full strings, never cut output
+.sp
+\fB\-c\fR: run the agent without color output
+\fB\-r\fR: run the agent with raw output
+\fB\-R\fR: run the agent in completely unparsed mode, with no return code of 1 in case of error\&. A little faster\&.
+\fB\-m\fR: run the agent with multiline output
+.sp
+\fB\-b\fR: run the agent on a specific bundle
+.sp
+\fB\-D\fR: define a class for this run
+.RE
+.PP
+\fBupdate\fR
+.RS 4
+update promises on agent\&. The agent will fetch the last version of its promises from its configured policy server\&.
+.sp
+\fBOptions\fR:
+.sp
+\fB\-i\fR: run the agent in information mode, prints basic information
+.sp
+\fB\-v\fR: run the agent in verbose mode, prints detailed information
+.sp
+\fB\-d\fR: run the agent in debug mode, prints low\-level information
+.sp
+\fB\-c\fR: run the agent without color output
+.RE
+.PP
+\fBversion\fR
+.RS 4
+get the agent version\&. Displays the version of the Rudder agent and of the underlying CFEngine agent\&.
+.RE
+.SS "remote"
+.sp
+commands for rudder remote, run with \fBrudder\fR \fBremote\fR \fIcommand\fR
+.PP
+\fBrun\fR
+.RS 4
+trigger the execution of a remote agent\&. This command allows to override the agent run schedule and to immediately update the promises and enforce them on th specified node\&. This command is currently only allowed from the policy server of the target node\&.
+.sp
+\fBArguments\fR:
+.sp
+\fBnode\fR: IP or hostname of the target node
+.sp
+\fBOptions\fR:
+.sp
+\fB\-i\fR: run the agent in information mode, prints basic information
+.sp
+\fB\-v\fR: run the agent in verbose mode, prints detailed information
+.sp
+\fB\-d\fR: run the agent in debug mode, prints low\-level information
+.sp
+\fB\-c\fR: run the agent without color output
+.RE
+.SS "server"
+.sp
+commands for rudder server, run with \fBrudder\fR \fBserver\fR \fIcommand\fR
+.PP
+\fBdebug\fR
+.RS 4
+run a debug
+\fIcf\-serverd\fR
+intended for a specific node\&. This command targets a specific node and does not affect the running infrastructure\&. It uses
+\fIiptables\fR
+to redirect the specific node communications to the port the debug server is listening on (5310 by default)\&.
+.sp
+Use Ctrl+C to stop the debug server\&.
+.sp
+\fBArguments\fR:
+.sp
+\fBnode\fR: IP or hostname of the host you want to debug
+.RE
+.SH "AUTHOR"
+.sp
+Normation SAS (contact@normation\&.com)
+.SH "RESOURCES"
+.sp
+Main web site: https://rudder\-project\&.org/
+.sp
+Sources of the rudder command\-line: https://github\&.com/Normation/rudder\-agent/
+.SH "COPYING"
+.sp
+Copyright (C) 2014\-2015 Normation SAS\&.
```

Comment:
Is this file supposed to be in this PR? I don't understand where it comes from, but it looks like it might be generated, and it's definitely not related to the ticket's title.